assets/js/util/sanitize.js

```diff
@@ -8,3 +8,25 @@ export const sanitizeHTML = ( unsafeHTML, domPurifyConfig = {} ) => {
 		__html: purify.sanitize( unsafeHTML, domPurifyConfig ),
 	};
 };
+
+/**
+ * Takes a URL string, removes the trailing stash if any and returns it.
+ *
+ * @since 1.18.0
+ * @private
+ *
+ * @param {string} url The URL with or without trailing slash.
+ * @return {string|null} The URL string after removing the trailing slash.
+ */
+export const unTrailingSlashIt = ( url ) => {
+	if ( typeof url !== 'string' ) {
+		return null;
+	}
+
+	if ( url.endsWith( '/' ) ) {
+		// remove the trailing slash
+		return url.slice( 0, -1 );
+	}
+
+	return url;
+};
```

Comment:
I would expect this to remove _all_ trailing slashes, not just the last one. The [WordPress function removes all slashes](https://developer.wordpress.org/reference/functions/untrailingslashit), so we should probably match that functionality.